src/main/kotlin/com/chloe/kotlinserv/HttpResponse.kt

```diff
@@ -0,0 +1,9 @@
+package com.chloe.kotlinserv
+
+typealias Header = Map<String, String>
+
+data class HttpResponse(
+    val code: String,
```

Comment:
http response code is Int in any case. You can create some object to have popular constant http codes